packages/grafana-ui/src/components/Table/Cells/JSONViewCell.tsx

```diff
@@ -25,7 +25,12 @@ export function JSONViewCell(props: TableCellProps): JSX.Element {
       value = JSON.parse(value);
     } catch {} // ignore errors
   } else {
-    displayValue = JSON.stringify(value, null, ' ');
+    try {
```

Comment:
Add to tableNG as well

Reply:
Next Gen table works fine, and I realized I removed the config value at some point to have it always on.

<img width="619" height="157" alt="Screenshot 2025-08-20 at 3 34 29 PM" src="https://github.com/user-attachments/assets/8c3e9e5b-ba67-4bee-b7ed-3ea424220bb8" />